python_modules/dagster/dagster/cli/api.py

```diff
@@ -323,6 +326,12 @@ def execute_step_command(input_json):
 
```

Comment:
if you do input_json there's actually an existing env var you can use? believe it's DAGSTER_CLI_API_EXECUTE_STEP_INPUT_JSON

(but that won't work if you gzip it so maybe not)